src/In.ProjectEKA.TMHHip/DataFlow/CustomWebclient.cs

```diff
@@ -0,0 +1,34 @@
+namespace In.ProjectEKA.TMHHip.DataFlow
+{
+    using System;
+    using System.Net;
+    using System.Security;
+
+    internal class CustomWebclient : WebClient
+    {
+        [SecuritySafeCritical]
+        private CustomWebclient() : base()
+        {
+        }
+
+        private readonly CookieContainer cookieContainer = new CookieContainer();
+
+
+        protected override WebRequest GetWebRequest(Uri myAddress)
+        {
+            WebRequest request = base.GetWebRequest(myAddress);
+            if (!(request is HttpWebRequest)) return request;
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4940156)